src/drawers/elementDrawer.ts

```diff
@@ -60,6 +60,15 @@ export module _Drawer {
     public _getSelector(): string {
       return this._svgElement;
     }
+
+    public _getPixelPoint(selection: D3.Selection, datum: any, index: number): Point {
+      switch(this._svgElement) {
+        case "circle":
+          return { x: parseFloat(selection.attr("cx")), y: parseFloat(selection.attr("cy")) };
```

Comment:
It seems like the better way to do this is to subclass `Drawer.Element` to create `Drawer.Circle`, which overrides this.

The `Drawer.Element` default implementation should ... hmm ... return some sort of `getBBox()` or `getBoundingClientRect` based implementation?